corehq/ex-submodules/casexml/apps/phone/data_providers/case/utils.py

```diff
@@ -17,7 +17,7 @@ def _get_required_updates(self):
         pairing. Should be a list of actions like [create, update, close]
         """
         ret = []
-        if not self.sync_token or not self.sync_token.phone_has_case(self.case.get_id):
+        if not self.sync_token or not self.sync_token.phone_is_holding_case(self.case.get_id):
```

Comment:
I think this is a bugfix. Should have been true even if the phone just had the dependent case (which it wouldn't have been with the previous code call)